nemo/collections/nlp/models/language_modeling/megatron_retro_model.py

```diff
@@ -99,6 +99,9 @@
     TransformerConfig = ApexGuardDefaults
 
     HAVE_MEGATRON_CORE = False
+    from typing import Any
+
+    RetroConfig = Any
```

Comment:
do we want to do the same for all of others mcore's classes we are trying to import under "try" block?
also, why don't use `ApexGuardDefaults`?

Reply:
> do we want to do the same for all of others mcore's classes we are trying to import under "try" block?

So this PR is intended to provide a pure pytorch/jit path for NeMo 2.0, so things are not guaranteed to work with NeMo 1.0 models (I have updated the description to reflect this). I added this change because previously, the import of the Retro model in https://github.com/NVIDIA/NeMo/blob/bdb4e89d9ac33d733f8ea7b21552628dda798825/nemo/collections/nlp/models/language_modeling/__init__.py was breaking the 2.0 codepath, but it looks like that's no longer the case anymore. If it makes things clearer, I go ahead and revert this change.